tests/base/quadrature_test.cc

```diff
@@ -176,7 +176,7 @@ check_faces(const std::vector<Quadrature<dim - 1> *> &quadratures,
           switch (dim)
             {
               case 2:
-                exact_int = 2 * (sub ? 2 : 1) / (double)(i + 1);
+                exact_int = 4 * (sub ? 2 : 1) / (double)(i + 1);
```

Comment:
One place where a recent patch created some merge conflicts was

https://github.com/dealii/dealii/blob/e5d72db95ba0dce251762435441241240d10368e/tests/base/quadrature_test.cc#L179-L189

i.e., #18306 made a similar fix (though to face quadratures, not subfaces).